mod.ts

```diff
@@ -266,7 +271,8 @@ const setGlobals = async (args: DenoliverOptions): Promise<void> => {
       before = args.after
     } else {
       try {
-        const path = posix.resolve(`${root}/${args.after}`)
+        var path = resolve(`${root}/${args.after}`)
+        if (Deno.build.os == 'windows') path = 'file://'.concat(path)
```

Comment:
I try to keep `var` and `let` out of the code whenever possible, and avoid mutation. I would suggest this approach instead.

```suggestion
        const path = Deno.build.os === "windows" ? 'file://'.concat(path) : resolve(`${root}/${args.after}`)
```